src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -111,17 +112,31 @@ function ReportActionItemSingle({
     const [innerPolicies] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {
         canBeMissing: true,
     });
+    const [iouActions] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${iouReport?.reportID}`, {
+        canBeMissing: true,
+        selector: (actions) => Object.values(actions ?? {}).filter(isMoneyRequestAction),
+    });
+
     const activePolicies = policies ?? innerPolicies;
     const policy = usePolicy(report?.policyID);
     const delegatePersonalDetails = action?.delegateAccountID ? personalDetails?.[action?.delegateAccountID] : undefined;
     const ownerAccountID = iouReport?.ownerAccountID ?? action?.childOwnerAccountID;
     const isReportPreviewAction = action?.actionName === CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW;
     const actorAccountID = getReportActionActorAccountID(action, iouReport, report, delegatePersonalDetails);
+
+    // We want to display only the sender's avatar next to the report preview if it only contains one person's expenses.
+    const areIOUActionsOnlyFromOnePerson = isReportPreviewAction && iouActions?.length && (iouActions.every(isActionCreator) || !iouActions.some(isActionCreator));
```

Comment:
NIT for better TS type inferring

```suggestion
    const areIOUActionsOnlyFromOnePerson = isReportPreviewAction && !!iouActions?.length && (iouActions.every(isActionCreator) || !iouActions.some(isActionCreator));
```

Reply:
Why do we check `isReportPreviewAction` here? Wouldn't that impact the accountID result for non-report action cases ?